jax-rs-error-handling-builder/src/test/java/com/ft/api/jaxrs/errors/RuntimeExceptionMapperTest.java

```diff
@@ -6,13 +6,11 @@
 import org.junit.Rule;
 import org.junit.Test;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.NotFoundException;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
```

Comment:
In the content programme we tend to discourage wildcard imports.

Reply:
And rightly so. I shall fix it.